### PR TITLE
chore: update gradle plugin to 1.6.0

### PR DESCRIPTION
**Pull request by adombeck**

### CODE REVIEW

**Comment by MarkusZoppelt** [comment]
> Are you aware of `go run ./tools/updater/updater.go --dependency gradle-plugin --version 1.6.0` ?

Yes, that's what I ran 👍🏼

File: test/projects/gradle/testsuite/build.gradle.kts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 plugins {
     id("java-library")
-    id("com.code-intelligence.cifuzz") version "1.4.0"
+    id("com.code-intelligence.cifuzz") version "1.5.0"
```
Comment (jochil) [inline test/projects/gradle/testsuite/build.gradle.kts:3]: shouldn't this be 1.6.0 as well?

Comment (MarkusZoppelt) [reply test/projects/gradle/testsuite/build.gradle.kts]: that file was missing in the updater list. It is in there now.

**Review by jochil — APPROVED** [review]

**Comment by adombeck** [comment]
@MarkusZoppelt you see how annoying those failing merge queues are, right?

**Pull request merged**